nonprofit-board-management.php

```diff
@@ -946,6 +951,39 @@ private function validate_gravatar( $user_email ) {
 
       return $has_valid_avatar;
     }
+
+    /**
+     * Check if the Block Editor is being used.
+     * 
+     * @return bool True if the Block Editor is being used, false if not.
+     */
+    private function is_block_editor_in_use() {
+
+      $use_block_editor = get_option( 'classic-editor-replace' ) === 'no-replace';
+
+      // Check if the Gutenberg plugin is installed and activated.
+      $gutenberg_plugin = ! ( false === has_filter( 'replace_editor', 'gutenberg_init' ) );
+
+      // Block editor is used by default since WordPress 5.0.
+      $block_editor = version_compare( $GLOBALS['wp_version'], '5.0-beta', '>' );
+
+      // If the Gutenberg plugin is not activated and the WordPress version is less than 5.0 we assume the Block Editor is not being used
+      if( ! $gutenberg_plugin && ! $block_editor ) {
+        return false;
+      }
+
+      // Check if the Classic Editor plugin is activated - the Classic Editor replaces the Block Editor
+      if( ! is_plugin_active( 'classic-editor/classic-editor.php' ) ) {
+        return true;
+      }
+
+      // Checks if the Classic Editor plugin is set to use the Block Editor but show a link to revert to the Classic Editor (Settings > Writing)
+      $use_block_editor = get_option( 'classic-editor-replace' ) === 'no-replace';
```

Comment:
If we are going to keep this is there any reason to have it in here twice? It looks like this code matches exactly to what's on line 962.

Reply:
I commented on the other line - this should stay the other should go.